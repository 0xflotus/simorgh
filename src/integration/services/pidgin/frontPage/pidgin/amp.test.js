/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service pidgin
 * @pathname /pidgin
 */

import runAmpTests from '../../../../pages/frontPage/ampTests';

describe('AMP pidgin frontPage', runAmpTests);
