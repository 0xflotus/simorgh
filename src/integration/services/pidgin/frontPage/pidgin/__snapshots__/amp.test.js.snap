// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP pidgin frontPage A11y I can see the skip to content link 1`] = `"Waka go wetin de inside"`;

exports[`AMP pidgin frontPage AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP pidgin frontPage AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP pidgin frontPage Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=70&p=pidgin.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:cps:52f3c7b1-83ed-2338-e050-17ac8045cd94]&x2=[amp]&x3=[news-pidgin]&x4=[pcm]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[index-home]&x8=[simorgh]&x9=[Domot+-+BBC+News+Pidgin]&x11=[1970-01-18T08:30:40.608Z]&x12=[1970-01-19T01:24:09.011Z]&ref=\${documentReferrer}",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP pidgin frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/pidgin"`;

exports[`AMP pidgin frontPage Footer I can see the footer copyright and external linking text 1`] = `"Â© 2020 BBC. De external site no concern BBC. De way wey we de take go external link."`;

exports[`AMP pidgin frontPage Header I can see the branding 1`] = `"BBC News, Pidgin"`;

exports[`AMP pidgin frontPage I can see Index Alsos 1`] = `"Another thing we de for inside dis tori"`;

exports[`AMP pidgin frontPage I can see sections 1`] = `"Top tori"`;

exports[`AMP pidgin frontPage I can see sections 2`] = `"Yarn Me Tori"`;

exports[`AMP pidgin frontPage I can see sections 3`] = `"Sport (click for more tori) See everitin"`;

exports[`AMP pidgin frontPage I can see sections 4`] = `"Ginger me"`;

exports[`AMP pidgin frontPage I can see sections 5`] = `"Informate me"`;

exports[`AMP pidgin frontPage I can see sections 6`] = `"Special reports 2"`;

exports[`AMP pidgin frontPage I can see sections 7`] = `"2018 World Cup"`;

exports[`AMP pidgin frontPage I can see sections 8`] = `"Podcast"`;

exports[`AMP pidgin frontPage I can see the headline 1`] = `"BBC News, Pidgin - Home"`;

exports[`AMP pidgin frontPage SEO Canonical link 1`] = `"http://localhost:7080/pidgin"`;

exports[`AMP pidgin frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP pidgin frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP pidgin frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP pidgin frontPage SEO Heading level 1 1`] = `"BBC News, Pidgin - Home"`;

exports[`AMP pidgin frontPage SEO Lang attribute 1`] = `"pcm"`;

exports[`AMP pidgin frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/pidgin",
    "@type": "WebPage",
    "name": "Domot",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
      "width": 1024,
    },
    "name": "BBC News Pidgin",
    "publishingPrinciples": "https://www.bbc.com/pidgin/institutional-48528766",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
  "url": "http://localhost:7080/pidgin",
}
`;

exports[`AMP pidgin frontPage SEO OG description 1`] = `"We dey give una latest tori on top politics, environment, business, sports, entertainment, health, fashion and all di oda things wey dey happen for West and Central Africa come add di rest of di world join. For better informate plus explanation of all di ogbonge tori wey pipo never hear about for inside West and Central Africa, BBC Pidgin dey serve am with video, audio, maps and oda graphics join."`;

exports[`AMP pidgin frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"`;

exports[`AMP pidgin frontPage SEO OG image alt 1`] = `"BBC News Pidgin"`;

exports[`AMP pidgin frontPage SEO OG locale 1`] = `"pcm"`;

exports[`AMP pidgin frontPage SEO OG site name 1`] = `"BBC News Pidgin"`;

exports[`AMP pidgin frontPage SEO OG title 1`] = `"Domot - BBC News Pidgin"`;

exports[`AMP pidgin frontPage SEO OG type 1`] = `"website"`;

exports[`AMP pidgin frontPage SEO OG url 1`] = `"http://localhost:7080/pidgin"`;

exports[`AMP pidgin frontPage SEO Page title 1`] = `"Domot - BBC News Pidgin"`;

exports[`AMP pidgin frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP pidgin frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP pidgin frontPage SEO Twitter creator 1`] = `"@BBCNews"`;

exports[`AMP pidgin frontPage SEO Twitter description 1`] = `"We dey give una latest tori on top politics, environment, business, sports, entertainment, health, fashion and all di oda things wey dey happen for West and Central Africa come add di rest of di world join. For better informate plus explanation of all di ogbonge tori wey pipo never hear about for inside West and Central Africa, BBC Pidgin dey serve am with video, audio, maps and oda graphics join."`;

exports[`AMP pidgin frontPage SEO Twitter image alt 1`] = `"BBC News Pidgin"`;

exports[`AMP pidgin frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"`;

exports[`AMP pidgin frontPage SEO Twitter site 1`] = `"@BBCNews"`;

exports[`AMP pidgin frontPage SEO Twitter title 1`] = `"Domot - BBC News Pidgin"`;

exports[`AMP pidgin frontPage Story Promo I can see a headline 1`] = `"AS E DE HAPPEN Senators don begin vote for Senate President"`;

exports[`AMP pidgin frontPage Story Promo I can see an image 1`] = `
<amp-img
  alt="Senator Ahmad Lawan and Ali Ndume"
  attribution="Other"
  height="549"
  layout="responsive"
  src="https://ichef.bbci.co.uk/news/660/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg"
  width="976"
/>
`;
